adoc/chapters/programming_interface.adoc

```diff
@@ -12417,9 +12428,7 @@ const property_list &propList = {})
       of the variable(s) described by [code]#vars# using the combination
```

Comment:
GitHub won't let me add a comment to the line I really want to comment upon, my comment is really on this line:

> Available only when [code]#has_known_identity<BinaryOperation, BufferT::value_type>::value# is [code]#false#.

I think you want to remove this line, and the lines similar below.  These descriptions correspond to the comments you removed in the synopsis.

Reply:
Good catch, thanks.  Fixed in a9faf3e98b49cee456a92890093f6eeff8fc0121.